pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -17,32 +17,28 @@
 import pytensor.tensor as pt
 import xarray
 from pymc.util import RandomState
-from pytensor.tensor import TensorVariable
 
-from pymc_marketing.clv.models.basic import CLVModel
+from pymc_marketing.clv.models import CLVModel
 from pymc_marketing.clv.utils import customer_lifetime_value, to_xarray
 
 
 class BaseGammaGammaModel(CLVModel):
     def distribution_customer_spend(
         self,
-        customer_id: np.ndarray | pd.Series,
-        mean_transaction_value: np.ndarray | pd.Series | TensorVariable,
-        frequency: np.ndarray | pd.Series | TensorVariable,
+        data: pd.DataFrame,
```

Comment:
Take a look at https://github.com/pymc-labs/pymc-marketing/pull/750 to see how to use object references, such as `~pandas.DataFrame`.